frontend/src/Album/Details/TrackRow.js

```diff
@@ -178,7 +178,6 @@ TrackRow.propTypes = {
   id: PropTypes.number.isRequired,
   albumId: PropTypes.number.isRequired,
   trackFileId: PropTypes.number,
-  monitored: PropTypes.bool.isRequired,
```

Comment:
Should we keep in track monitor field in case we decide to support selective track saving (only keep certain tracks from a album), or support a service that downloads by track?

Reply:
As with the `Album` tags, my view is that we should remove it because it adds complication and is likely to break given it's not in use.  It can always be reinstated when it's needed.  Again, happy to leave it if you prefer.

Reply:
I’m good with that